resources/LookDev/emptyLook.mtlx

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<!-- Test the RtFileIo can read look where all optional elements are missing,
+	 simulating re-reading a file with unconnected Rt nodes. -->
+<materialx version="1.38">
+  <look name="emptyLook">
+  </look>
+  <look name="emptyMatAssign">
+    <materialassign name="defaultMaterialAssign" material=""/>
```

Comment:
I think this should fail Document::validate() and throw an exception there. Could be we're not validating on read?
If it's called and it's not failing we should patch this up.

Reply:
There is value in being able to re-read a work in progress. 
Is is valid for rendering? Definitely not. 
But it is valid enough for opening in the the node-editor.

Reply:
I guess my point is, is this a valid document -- and it is not. A material assignment to nothing has no meaning.